src/adaptix/_internal/morphing/load_error.py

```diff
@@ -130,6 +130,14 @@ class BadVariantError(LoadError):
     input_value: Any
 
 
+@custom_exception
+@dataclass(eq=False)
+class MultipleBadVariantError(LoadError):
+    allowed_values: Iterable[Any]
+    input_values: Iterable[Any]
```

Comment:
All ``LoadError`` must have ``input_value: Any`` field (not ``input_values``). Maybe for this case it can have ``Iterable[Any]`` type